x/group/internal/math/math.go

```diff
@@ -0,0 +1,35 @@
+// Package math provides helper functions for doing mathematical calculations and parsing for the ecocredit module.
+package math
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/apd/v2"
+)
+
+var exactContext = apd.Context{
```

Comment:
why the name `exactContext`?

Reply:
I believe it's because we are having arbitrary precision here instead of confining it to a particular value.